From e15403f77df601739cd13130d02f52cd452df01f Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Thu, 30 Jun 2022 14:18:51 +0300
Subject: [PATCH] [SW-93513] gaudi2: rename erom pcie functions

The hl_pcie_init and preboot_pcie_enable functions are
defined in erom_pcie.c and also in ppboot.c. Therefore for
better code readablility renaming these functions in erom_ext
application.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Ia7d528d27c82a1cf8d1f02678ef9a62ed6df7575
(cherry picked from commit 3851932a83c3968e82c13792c0d67610f6e38b32)
---

diff --git a/app/ext_rom/erom_pcie.c b/app/ext_rom/erom_pcie.c
index 93b4956..3008bf7 100644
--- a/app/ext_rom/erom_pcie.c
+++ b/app/ext_rom/erom_pcie.c
@@ -112,7 +112,7 @@
 	regmap_writel(3, mmPCIE_AUX_LTSSM_EN);
 }
 
-void preboot_pcie_enable(void)
+void erom_pcie_enable(void)
 {
 	u32 rst_src;
 
@@ -152,7 +152,7 @@
 	return false;
 }
 
-void hl_pcie_init(void)
+void erom_pcie_init(void)
 {
 	if (hl_pcie_link_up()) {
 		hl_pcie_enable_access();
@@ -268,5 +268,5 @@
 
 	cpu_boot_if_init();
 	hl_plat_early_init();
-	hl_pcie_init();
+	erom_pcie_init();
 }
